pkgs/development/python-modules/ring_doorbell/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+, pytestCheckHook
+, pytz
+, requests
+, requests-mock
+, requests_oauthlib
+}:
+
+buildPythonPackage rec {
+  pname = "ring_doorbell";
+  version = "0.6.2";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0q35sji2f4lphlr0hdpzazfbi6rrnyadh1k5q9abicr759r3gmgv";
+  };
+
+  propagatedBuildInputs = [
+    pytz
+    requests
+    requests_oauthlib
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    requests-mock
+  ];
+
+  pythonImportsCheck = [ "ring_doorbell" ];
+
+  meta = with lib; {
+    homepage = "https://github.com/tchellomello/python-ring-doorbell";
+    description = "A Python library to communicate with Ring Door Bell (https://ring.com/)";
+    license = licenses.lgpl2;
```

The license tag in the `setup.py` file says [LGPLv2+](https://github.com/tchellomello/python-ring-doorbell/blob/master/setup.py#L23).

```suggestion
    license = licenses.lgpl2Plus;
```

---

It actually says LGPL v3+, but the actual LICENSE file is v2 so I think what you suggest makes sense.

---

You should file an upstream pull request to fix that.

---

I filed an issue since I'm not sure which version they're actually intending to use: https://github.com/tchellomello/python-ring-doorbell/issues/241.

---

> It actually says LGPL v3+...

Right

---

Upstream has fixed their LICENSE file, so updated to `lgpl3Plus`.